app/components/weather-selector.js

```diff
@@ -2,8 +2,6 @@ import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 
 export default class WeatherSelector extends Component {
-  @tracked currentWeather = "snowy";
```

Comment:
seems like this wasn't necessary so I deleted it

Reply:
isnt' that used in the template?

Reply:
uh yeah woops
can the template reference the current weather stored in the service instead?

Reply:
yeah you could inject the service and do it that way instead